docs/explanations/architecture/styles.md

```diff
@@ -4,17 +4,15 @@ This document introduces the main concepts related to styles that affect the use
 
 1. HTML and CSS
 2. Block styles
-
 -   From UI controls to HTML markup
 -   Block Supports API
 -   Current limits of the Block Supports API
-
 3. Global styles
-
 -   Gather data
 -   Consolidate data
 -   From data to styles
 -   Current limits of the Global styles API
+4. Structural layout styles
```

Comment:
Oh, thanks, that tool's super helpful! I'll update the TOC in this PR since it very much improves navigability of this pretty long document.

> I took it for a test spin using this markdown and I also noticed that the headings don't start from # (h1)

I double-checked, and I believe we're starting at `h2` because in the handbook, the page title gets the `h1` element:

<img width="670" alt="image" src="https://user-images.githubusercontent.com/14988353/193981070-512bd5df-bf85-4b06-b899-df9b853a5975.png">

Reply:
Updated the TOC.

Reply:
> the page title gets the h1 element:

Good call! Thanks for the heads up.